src/test/java/org/broadinstitute/hellbender/tools/copynumber/GermlineCNVCallerIntegrationTest.java

```diff
@@ -88,11 +105,67 @@ public void testCaseWithHiddenArguments() {
         runCommandLine(argsBuilder);
     }
 
-    @Test(groups = {"python"}, enabled = false)
+    /**
+     * Test that result falls within a numerical accuracy bound defined. This test is meant to detect things like major
+     * Python library updates that affect gCNV results, and detect unintentional consequences of minor model changes.
+     */
+    @Test(groups = {"python"})
+    public void testNumericalAccuracy() {
+        final ArgumentsBuilder argsBuilder = new ArgumentsBuilder();
+        final String outputPrefix = "test-germline-cnv-accuracy";
+        Arrays.stream(TEST_COUNT_FILES, 0, 20).forEach(argsBuilder::addInput);
+        argsBuilder.add(GermlineCNVCaller.RUN_MODE_LONG_NAME, GermlineCNVCaller.RunMode.COHORT.name())
+                .add("L", SIM_INTERVAL_LIST_SHARD_0)
+                .add(IntervalArgumentCollection.INTERVAL_MERGING_RULE_LONG_NAME, IntervalMergingRule.OVERLAPPING_ONLY.toString())
+                .add(GermlineCNVCaller.CONTIG_PLOIDY_CALLS_DIRECTORY_LONG_NAME,
+                        CONTIG_PLOIDY_CALLS_OUTPUT_DIR.getAbsolutePath())
+                .add(StandardArgumentDefinitions.OUTPUT_LONG_NAME, OUTPUT_DIR.getAbsolutePath())
+                .add(CopyNumberStandardArgument.OUTPUT_PREFIX_LONG_NAME, outputPrefix);
+        runCommandLine(argsBuilder);
+        MODEL_FILES_TO_COMPARE.forEach(f -> CopyNumberTestUtils.assertFilesEqualUpToAllowedDeltaForDoubleValues(
+                new File(Paths.get(OUTPUT_DIR.getAbsolutePath(), outputPrefix + "-model").toString(), f),
+                new File(MODEL_EXACT_MATCH_EXPECTED_SUB_DIR,  f),
```

Comment:
Note that I reused a previosly computed gCNV model for the numerical accuracy test. Good news is that the results haven't deviated since the time we made these test files. However, perhaps we should regenerate and create dedicated model files just for this test - especially to have a more stable test that compares denoised copy ratio values (see comment about the mu_denoised_copy_ratio_t.tsv).

Reply:
Yes let's do what @samuelklee suggested and have a `UPDATE_EXACT_MATCH_EXPECTED_OUTPUTS` option.

Reply:
I've implemented that. Note that I had to move the test files into the new directory 'gcnv-numerical-accuracy'.